R/authors-at-r.R

```diff
@@ -145,15 +156,42 @@ idesc_add_role <- function(self, private, role, given, family, email,
   self$set_authors(orig)
 }
 
+idesc_add_orcid <- function(self, private, orcid, given, family, email,
+                            comment, role) {
+
+  check_author_args(given = given, family = family,
+                    email = email,
+                    comment = comment,
+                    orcid = orcid, role = role)
+
+  orig <- idesc_get_authors(self, private, ensure = FALSE)
+  wh <- search_for_author(
+    orig, given = given, family = family, email = email, comment = comment,
+    orcid = NULL,
+    role = role
+  )
+
+  for (w in wh$index) {
+    orig <- set_author_field(
+      orig,
+      w,
+      "comment",
+      c(comment, ORCID = orcid)
```

Comment:
So, what happens if there  is already an orcid  there?

Reply:
Right. Do you agree that the old one should get replaced by the new one if that's the case?

Reply:
Yes, I do. It would be nice to have a test case for this as well. Thanks!

Reply:
I'll also add a test for this.